code/_globalvars/logging.dm

```diff
@@ -43,6 +43,9 @@ GLOBAL_PROTECT(tgui_log)
 GLOBAL_VAR(world_shuttle_log)
 GLOBAL_PROTECT(world_shuttle_log)
 
+GLOBAL_VAR(subsystem_log)
+GLOBAL_PROTECT(subsystem_log)

```

Comment:
Let's not add this here unless you're going to refactor all existing subsystems to use it.